config/routes.rb

```diff
@@ -1,14 +1,21 @@
 Rails.application.routes.draw do
+  get 'posts/new'
   get 'posts/index'
+  get 'comments/create'
   get 'posts/show'
   get 'users/index'
   get 'users/show'
+  get 'likes/create'
```

Comment:
- [ ] Kindly delete these paths as you have already defined them using resources.